pkg/controller/controllercmd/builder.go

```diff
@@ -190,35 +201,51 @@ func (b *ControllerBuilder) WithEventRecorderOptions(options record.CorrelatorOp
 
 // Run starts your controller for you.  It uses leader election if you asked, otherwise it directly calls you
 func (b *ControllerBuilder) Run(ctx context.Context, config *unstructured.Unstructured) error {
-	clientConfig, err := b.getClientConfig()
-	if err != nil {
-		return err
+	var clientConfig *rest.Config
+
+	if b.kubeAPIServerConfigFile != nil {
```

Comment:
this change appears to have broken the automatic in-cluster configuration getting of a client.